src/test/java/edu/ucla/library/prl/harvester/services/HarvestJobSchedulerServiceIT.java

```diff
@@ -164,7 +163,7 @@ public void tearDown(final Vertx aVertx, final VertxTestContext aContext) {
      * @param aContext A test context
      */
     @Test
-    @Timeout(value = 90, timeUnit = TimeUnit.SECONDS)
+    @Timeout(value = 10, timeUnit = TimeUnit.SECONDS)
```

Comment:
If the default timeout is 30 seconds, do you need to specify a lower number?

Reply:
"Fail _really_ fast"!

Seriously though: I think it helps convey helpful info about the expected behavior of the test, since that specification involves elapsed time.